security-jwt/src/main/java/io/micronaut/security/token/jwt/validator/NotBeforeJwtClaimsValidator.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.security.token.jwt.validator;
+
+import java.util.Date;
+
+import javax.inject.Singleton;
+
+import com.nimbusds.jwt.JWTClaimsSet;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.util.StringUtils;
+import io.micronaut.security.token.jwt.generator.claims.JwtClaims;
+
+/**
+ * Validate current time is not before the not-before claim of a JWT token
+ *
+ * @author Jason Schindler
+ * @since 2.4.0
+ */
+@Singleton
+@Requires(property = NotBeforeJwtClaimsValidator.NOT_BEFORE_PROP, value = StringUtils.TRUE)
```

Comment:
I made this require the property to be true instead of enabling it by default because I presumed that enabling it by default would constitute a breaking change.